api/src/test/java/org/openmrs/module/pihcore/search/PihPatientSearchAlgorithmTest.java

```diff
@@ -13,13 +13,17 @@
 import org.openmrs.module.pihcore.metadata.Metadata;
 import org.openmrs.module.registrationcore.api.search.PatientAndMatchQuality;
 import org.springframework.beans.factory.annotation.Autowired;
+import org.testcontainers.shaded.org.apache.commons.lang3.ObjectUtils;
```

Comment:
Not a blocker, but this is a weird library I haven't seen before?

Reply:
Yeah, can you fix that @cioan ?  I assume it was just included inadvertently when you were going for a standard commons lang import?

Reply:
Yes, thank you! I have removed that unused library. Thanks!